components/subscribe-form.tsx

```diff
@@ -110,7 +110,9 @@ const SubscribeForm: FC = () => {
             Sign up for dao updates
           </Button>
           <FormControl hidden={status !== 'unsubmitted'}>
-            <FormHelperText>We will never share your email.</FormHelperText>
+            <FormHelperText color="gray.600">
```

Comment:
might look off in dark mode but really happy someone else is familiar with chakra!

Reply:
😅  I just looked up the docs. At my main job we use styled components :).

Reply:
i believe this is styled-components under the hood (tho i think there  might be another layer in there).